src/main/docker/app.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
```

Comment:
Among all your modification messages, it's the only that I'm not sure to know how to resolve.